### PR TITLE
[wip] Add/publicize scheduling

### PR DESCRIPTION
**Pull request by retrofox**

### CODE REVIEW

File: client/my-sites/post-share/index.jsx [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+
+		const classes = classNames( 'post-share__wrapper', {
+			'has-connections': this.hasConnections()
+		} );
+
+		return (
+			<div className="post-share">
+				{ this.props.requesting &&
+					<Notice status="is-warning" showDismiss={ false }>
+							{ this.props.translate( 'Sharing…' ) }
```
Comment (a8ci18n) [inline client/my-sites/post-share/index.jsx]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Sharing...' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/publicize-scheduling)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (yoavf) [inline client/my-sites/post-share/index.jsx]: `…` > `...`

The existing string is [here](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/posts/post-share.jsx#L149), so if we can change that one before committing this one, it'll be better.

File: client/my-sites/post-share/index.jsx [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+
+		if ( ! siteId || ! postId ) {
+			return null;
+		}
+
+		return (
+			<div className="post-share">
+				{ requesting &&
+					<Notice status="is-warning" showDismiss={ false }>
+							{ translate( 'Sharing…' ) }
```
Comment (a8ci18n) [inline client/my-sites/post-share/index.jsx]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Sharing...' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/publicize-scheduling)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/post-share/index.jsx [hunk 3]
```diff
@@ -0,0 +1,436 @@
...
+						</Notice>
+				}
+
+				<div className={ classes }>
+					<QueryPublicizeConnections siteId={ siteId } />
+					<QueryPostTypes siteId={ siteId } />
+
+					<div className="post-share__head">
+						<h4 className="post-share__title">
+							{ translate( 'Share this post' ) }
```
Comment (a8ci18n) [inline client/my-sites/post-share/index.jsx]: Hi! I've found a possible matching string that has already been translated **7** times:
`translate( 'Share this post!' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/post-share/index.jsx [hunk 4]
```diff
@@ -0,0 +1,436 @@
...
+						comment: '%s will be replaced by a formatted price, i.e $9.99'
+					} )
+				}
+				description={ translate( 'Live chat support and no advertising.' ) }
+				list={ [
+					translate( 'Live chat support' ),
+					translate( 'No Advertising' )
+				] }
+				plan={ PLAN_BUSINESS }
+				title={ translate( 'Upgrade to a Business Plan!' ) }
```
Comment (a8ci18n) [inline client/my-sites/post-share/index.jsx]: Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'To upload themes, upgrade to Business Plan' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by retrofox** [comment]
Close it in favor of #12471

**Pull request closed without merging**